src/dialogs/more-info/controls/more-info-cover.js

```diff
@@ -21,7 +21,6 @@ class MoreInfoCover extends LocalizeMixin(PolymerElement) {
         .current_position,
         .tilt {
           max-height: 0px;
-          overflow: hidden;
```

Comment:
Now it will not be hidden when it should?

Reply:
You are correct. I just now had my cover pretend that it cannot tilt, and then this simplistic fix does not work.

Probably better to somehow fix that then inside the ha-slider. But snce I am not a CSS wiz, I guess that might be beyond my capabilities.